samples/asgardeo-react-app/.env

```diff
@@ -1,4 +1,4 @@
-PORT=3000
-HOST="localhost"
-DISABLE_DEV_SERVER_HOST_CHECK=false
-HTTPS=true
+VITE_PORT=3000
+VITE_HOST="localhost"
+VITE_DISABLE_DEV_SERVER_HOST_CHECK=false
```

Comment:
I think we don't need this for vite. Could u please check?

Reply:
Done